docs/api/modules/App-Configuration.md

```diff
@@ -170,6 +171,42 @@ In practice, this allows POST requests from given origins to return partially re
 * [Frank Lloyd Root's `appConfig`](../../../prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/config.js)
 * In Practice: [Partial Rendering](../../../README.md#partial-rendering)
 
+## `pwa`
+**Module Type**
+* ✅ Root Module
+* 🚫 Child Module
+
```

Comment:
Could we add a description with links to outside documentation for this? (e.g. `configureRequestLog` entry father down this document)

Reply:
I added more info section and very brief description. I will certainly revisit this and elaborate.

Reply:
I've updated!